src/_gettsim/transfers/erwerbsm_rente.py

```diff
@@ -0,0 +1,224 @@
+    """Checking Rentenfaktor for Erwerbsminderungsrente
+    (pension for reduced earning capacity)
+    based on input variable teilw_erwerbsm_rente
+
+    Als Input Variable
+    Parameters
+    ----------
+    teilw_erwerbsm_rente
+        See basic input variable :ref:`teilw_erwerbsm_rente <teilw_erwerbsm_rente>.
+
+    Returns
+    -------
+    Rentenfaktor
+
+    """
+
+    if teilw_erwerbsm_rente:
+        out = 0.5
+
+    else:
+        out = 1
+
+    return out
+
+
+def erwerbsm_rente_vor_grundr_m(
+    erwerbsm_rente_zugangsfaktor: float,
+    entgeltp_erwerbsm_rente: float,
+    rentenwert: float,
+    rentenfaktor_erwerbsm_rente: float,
+    anspruch_erwerbsm_rente: bool,
+) -> float:
+    """Calculating the Erwerbsminderungsrente pension claim
+
+    Parameters
+    ----------
+    ges_rente_zugangsfaktor
+        See :func:`ges_rente_zugangsfaktor`.
+    entgeltp_fuer_erwerbsm_rente
+        See :func:`entgeltp_fuer_erwerbsm_rente`.
+    rentenwert
+        See :func:`rentenwert`.
+    rentenfaktor_fuer_erwerbsm_rente
+        See :func:`rentenfaktor_fuer_erwerbsm_rente`.
+    rentner
+        See basic input variable :ref:`rentner <rentner>.
+    Returns
+    -------
+    Erwerbsminderungsrente pension claim
+
+    """
+
+    if anspruch_erwerbsm_rente:
+        out = (
+            entgeltp_erwerbsm_rente
+            * erwerbsm_rente_zugangsfaktor
+            * rentenwert
+            * rentenfaktor_erwerbsm_rente
+        )
+    else:
+        out = 0.0
+
+    return out
+
+
+def erwerbsm_rente_altersgrenze(
+    erwerbsm_rente_params: dict,
+) -> float:
+    """Calculating Altersgrenze for Erwerbsminderungsrente
+    (pension for reduced earning capacity)
+    Parameters
+    ----------
+    erwerbsm_rente_params
+        See params documentation :ref:`erwerbsm_rente_params <erwerbsm_rente_params>.
+
+    Returns
+    -------
+    Altersgrenze for Erwerbsminderungsrente
+
+    """
+
+    out = erwerbsm_rente_params["altersgrenze_abschlagsfrei_in_m"] / 12
+
+    return out
+
+
+def erwerbsm_rente_zugangsfaktor(
+    jahr_renteneintr: int,
+    geburtsjahr: int,
+    erwerbsm_rente_altersgrenze: float,
+    ges_rente_params: dict,
+    erwerbsm_rente_params: dict,
+) -> float:
+    """Calculating Zugangsfaktor for Erwerbsminderungsrente
+    (pension for reduced earning capacity)
+
+    Parameters
+    ----------
+    jahr_renteneintr
+        See basic input variable :ref:`jahr_renteneintr <jahr_renteneintr>.
+    geburtsjahr
+        See basic input variable :ref:`geburtsjahr <geburtsjahr>.
+    erwerbsm_rente_altersgrenze
+        See :func:`erwerbsm_rente_altersgrenze`
+    ges_rente_params
+        See params documentation :ref:`ges_rente_params <ges_rente_params>.
+    erwerbsm_rente_params
+        See params documentation :ref:`erwerbsm_rente_params <erwerbsm_rente_params>.
+
+    Returns
+    -------
+    Zugangsfaktor for Erwerbsminderungsrente (pension for reduced earning capacity)
+
+    """
+    zugangsfaktor = (
```

Comment:
I decided for the yearly basis because gettsim is doing it that way for the normal pension and so I could use the same parameter -> ges_rente_params["zugangsfaktor_veränderung_pro_jahr“]. But I think it would be no big deal to change that.

Reply:
Should not matter for precision (or is it fixed to integers?) and implementation should be as close as possible to the law / regulations.

Reply:
That's fine then, thank you for the answers!